src/Console/Output/ProcessOutput.php

```diff
@@ -91,11 +93,21 @@ public function __destruct()
 
     public function onFixerFileProcessed(FixerFileProcessedEvent $event)
     {
+        if (
+            null === $this->files
+            && null !== $this->symbolsPerLine
+            && 0 === $this->processedFiles % $this->symbolsPerLine
+            && 0 !== $this->processedFiles
+        ) {
+            $this->output->writeln('');
```

Comment:
I want to have as close to v3 version as possible, yet having all v2 progress types available to achieve.
if you say that it's possible to (internally) call the service with parameters that would lead to this case, i would either allow that (current state) with TODO (as requested), or disallow that by throwing exception